docs/user-event/api-utility.mdx

```diff
@@ -0,0 +1,179 @@
+---
+id: utility
+title: Utility APIs
+---
+
+The following APIs don't have one-to-one equivalents in a real user
+interaction.  
+Their behavior is therefore an interpretation how the "perceived" user
+interaction might be translated to actual events on the DOM.
+
+## clear()
+
+```ts
+clear(element: Element): Promise<void>
+```
+
+This API can be used to easily clear an editable element.
+
+1. Focus element
+1. Select all contents as per browser menu
+1. Delete contents as per browser menu
+
+```jsx
+test('clear', async () => {
+  render(<textarea value="Hello, World!" />)
+
```

Comment:
I think we can reduce the line breaks here :)

Reply:
imho the line breaks between arrange-act-assert make it more readable and maybe make the test patterns more easily digestible for newcomers because it makes it easier to spot the same pattern in short as well as in longer tests.

(White space is cheap. No reason to be stingy with it. ^^)

Reply:
No worries, this sounds like a good reason :)